eventuals/errors.h

```diff
@@ -12,11 +12,13 @@ namespace eventuals {
 // All errors should derive from 'Error' and be copyable
 // since 'std::make_exception_ptr' works with a copy argument
 // and there is no way to move error in it.
-class Error {
+class Error : public std::exception {
  public:
   virtual ~Error() = default;
 
-  virtual std::string what() const noexcept = 0;
+  const char* what() const noexcept override {
+    return "Eventuals inner 'Error'";
```

Comment:
No need to override this! The subclasses will, and if they forget to you want that to be a compile time error!